client/src/Dashboard/Dashboard.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import '../index';
+import {BoldBI} from '@boldbi/boldbi-embedded-sdk';
+
+
+//NodeJs application would be run on http://localhost:8080, which needs to be set as `apiHost`
```

Comment:
//NodeJs application will run on http://localhost:8080, which needs to be set as `apiHost.`

Reply:
updated this line